TestAll.ps1

```diff
@@ -0,0 +1,110 @@
+#Requires -RunAsAdministrator
```

Comment:
Originally this script was running DevCheck which has to be in Admin mode, but since this is only triggering the tests now, we can remove.

BUT it is a directive that only allows the script to run as admin.

Reply:
Looks like dism needs elevation.

Reply:
Deleting this and moving DisplayInfo to RunTestSteps.yml

Reply:
DevCheck doesn't require admin
_Some operations_ in DevCheck require admin

Adding+Removing certs and registering/starting/stopping services

Search for "Get-IsAdmin" in DevCheck.ps1 for what needs admin

Reply:
Since TestAll isn't doing any of those operations, they're all preinstalled for us, we can remove admin requirement.